cmd/config/config.go

```diff
@@ -644,6 +644,15 @@ func (c Config) SetKVS(s string, defaultKVS map[string]KVS) error {
 		currKVS.Set(Comment, v)
 	}
 
+	hkvs := HelpSubSysMap[subSys]
+	for _, hkv := range hkvs {
+		v, _ := currKVS.Lookup(hkv.Key)
+		if v == "" && !hkv.Optional {
```

Comment:
This check doesn't allow for a valid value to be an empty string

Reply:
Yes there are no valid empty strings @krisis if it's mandatory

Reply:
Every requires a value, it cannot be empty. If it's empty it means it's optional.